slack-message-sender.el

```diff
@@ -45,9 +45,50 @@
         (setq message-id 1)
       (cl-incf message-id))))
 
+(defun slack-escape-message (message)
```

Comment:
when using `slack-message-embed-mention` or `slack-message-embed-channel`, this function escapes control sequence and these two functions don't work. 
I'm very glad if you fix these two functions just return user-name or channel name.